src/commands/game_options/release.ts

```diff
@@ -34,6 +39,33 @@ export default class ReleaseCommand implements BaseCommand {
         ],
     };
 
+    slashCommands = (): Array<Eris.ChatInputApplicationCommandStructure> => [
+        {
+            name: "release",
+            description: LocalizationManager.localizer.translateByLocale(
+                LocaleType.EN,
+                "command.release.help.interaction.description"
+            ),
+            type: Eris.Constants.ApplicationCommandTypes.CHAT_INPUT,
+            options: [
+                {
+                    name: "release",
+                    description:
+                        LocalizationManager.localizer.translateByLocale(
+                            LocaleType.EN,
+                            "command.release.help.interaction.description"
+                        ),
+                    type: Eris.Constants.ApplicationCommandOptionTypes.STRING,
+                    required: false,
+                    choices: Object.values(ReleaseType).map((releaseType) => ({
+                        name: releaseType,
+                        value: releaseType,
```

Comment:
Looking back at this, should we add a description for what each choice means here?